src/Processors/Transforms/JoiningTransform.h

```diff
@@ -8,21 +8,63 @@ namespace DB
 class IJoin;
 using JoinPtr = std::shared_ptr<IJoin>;
 
-class JoiningTransform : public ISimpleTransform
+class IBlockInputStream;
+using BlockInputStreamPtr = std::shared_ptr<IBlockInputStream>;
+
+/// Join rows to chunk form left table.
+/// This transform usually has two input ports and one output.
+/// First input is for data from left table.
+/// Second input has empty header and is connected with AddingJoined.
```

Comment:
AddingJoined looks like class name, but I have not found some entity with such name. What does it refer to?

Reply:
It's renames to `FillingRightJoinSideTransform`

Reply:
fixed